announcements.py

```diff
@@ -36,32 +50,37 @@ def run(self):
             time.sleep(sleep_time_seconds + 1) # Offset by 1 second
 
             # POST Request to send message
-            # url = 'https://your.endpoint.here'
+            current_day = datetime.datetime.now().strftime('%A').lower()
+            print("[" + str(datetime.datetime.now()) + "] DEBUG - Announcement (id: " + str(self.id) + ") post on " + current_day + ": " + str(self.selected_days[current_day]))
+            if self.selected_days[current_day]:
+                url = 'https://your.endpoint.here'
 
-            # headers = {
-            #     # Already added when you pass json= but not when you pass data=
-            #     # 'Content-Type': 'application/json'
-            # }
+                headers = {
+                    # Already added when you pass json= but not when you pass data=
+                    # 'Content-Type': 'application/json'
+                }
 
-            # json_data = {
-            #     'text': self.text
-            # }
+                json_data = {
+                    'text': self.text
+                }
 
-            # requests.post(url, headers=headers, json=json_data, verify=False)
-            print("[" + str(datetime.datetime.now()) + "] INFO - Request sent with message: " + self.text)
+                # requests.post(url, headers=headers, json=json_data, verify=False)
```

Comment:
Think you may have forgot about this one.

Reply:
Fixed